en/docs/reference/mediators/property-reference/http-transport-properties.md

```diff
@@ -169,7 +169,7 @@ appending a context to the target URL in RESTful invocations.
 <li>check if an outgoing request/response generated from the mediation flow has an entity body or not</li>
 </ul>
 <b>Note</b>:
-<p>If using the <a href="../../../../references/mediators/payloadFactory-Mediator">PayloadFactory mediator</a>, this property does not need to be manually set since it is done automatically by the mediator.</p>
+<p>If using the <a href="{{base_path}}/references/mediators/payloadfactory-mediator">PayloadFactory mediator</a>, this property does not need to be manually set since it is done automatically by the mediator.</p>
```

Comment:
```suggestion
<p>If using the <a href="{{base_path}}/reference/mediators/payloadfactory-mediator">PayloadFactory mediator</a>, this property does not need to be manually set since it is done automatically by the mediator.</p>
```